pkg/rpcclient/nep17.go

```diff
@@ -1,188 +0,0 @@
-package rpcclient
-
-import (
-	"fmt"
-
-	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
-	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
-	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
-	"github.com/nspcc-dev/neo-go/pkg/util"
-	"github.com/nspcc-dev/neo-go/pkg/wallet"
-)
-
-// TransferTarget represents target address, token amount and data for transfer.
-type TransferTarget struct {
```

Comment:
NeoFS adm needs to be updated. And they still use custom code to create multitransfer script, this can be refactored as far.

Reply:
https://github.com/nspcc-dev/neofs-node/blob/100660942423cc75f4ba057916efac4ab2dd47ef/cmd/neofs-adm/internal/modules/morph/initialize_transfer.go#L36 and the related code.

Reply:
Yeah, it should feel the pain of using deprecated things.